src/Reflection/DeprecatableReflection.php

```diff
@@ -7,4 +7,6 @@ interface DeprecatableReflection
 
 	public function isDeprecated(): bool;
 
+	public function getDeprecatedDescription(): ?string;
```

Comment:
This is a problem - if we add it to this interface, it can be released with 0.12. If you remove it from the interface, it can be merged immediately.

Reply:
So, should I remove from the interface, but keep implementation on the methods, and document somewhere "add to interface"?

Reply:
Yes, thank you. I'll have it in my todolist for 0.12.

Reply:
:) Cool, just wanted to make sure